website/hw/hw8.rst

```diff
@@ -638,8 +768,8 @@ Now, from your pipeline’s page, do the following:
    of the page.
```

Comment:
Very minor, but the page is now a "Review Apps" section where the actual button is "Enable"

Reply:
Is the button actually "Enable"? It shows up as "Enable Review Apps" on my end.

![Screenshot from 2021-05-15 10-32-02](https://user-images.githubusercontent.com/448918/118369421-e9487c00-b568-11eb-9a3f-38532205b982.png)

Reply:
@borjasotomayor Ah, I see. I was on the settings page. From the pipeline homepage, I see exactly what's in the image above.